Jenkinsfile

```diff
@@ -78,9 +78,7 @@ if (env.CHANGE_TITLE && env.CHANGE_TITLE.contains('[ci-skip]')) {
                                 checkout scm
                                 sh 'echo -n $(git rev-parse HEAD) > go/revision'
                                 sh "git add go/revision"
-                                env.GIT_COMMITTER_NAME = 'Jenkins'
-                                env.GIT_COMMITTER_EMAIL = 'ci@keybase.io'
-                                sh 'git commit --author="Jenkins <ci@keybase.io>" -am "revision file added"'
+                                sh "git stash"
```

Comment:
also, [these](https://github.com/keybase/client/blob/992d880cd07cb5b481b61dd29592ab60a9fe1120/Jenkinsfile#L167-L169) lines need to move out of the `parallel()` block, to after [here](https://github.com/keybase/client/blob/992d880cd07cb5b481b61dd29592ab60a9fe1120/Jenkinsfile#L289)